Resources/keybinds.yml

```diff
@@ -255,6 +255,10 @@ binds:
   type: State
   key: V
   mod1: Shift
+- function: OfferItem
+  type: State
+  key: V
```

Comment:
I don't know what a good bind would be, but Ctrl+V is wild

Reply:
now F, I hope it will be better this way.